engines/python/setup/djl_python/input_parser.py

```diff
@@ -148,10 +148,6 @@ def parse_text_inputs_params(request_input: TextInput, input_item: Input,
 
     request_input.input_text = inputs
     request_input.parameters = param
-    # assigns input_ids
-    # TODO: for dynamic batching, or HF pipeline, tokenizer is applied differently.
-    if kwargs.get("tokenizer") and kwargs.get("is_rolling_batch"):
-        request_input.input_ids = tokenizer.encode(request_input.input_text)
```

Comment:
Do we need the tokenizer at all anymore in this function?

Reply:
It is needed for chat_completions and bedrock use-cases to apply the chat-template.